app/views/home.slim

```diff
@@ -16,6 +17,8 @@ section.border.rounded.homepage
 
     small.form-text.text-muted.mb-2 We'll find and load your inbox for you once you press enter
 
+    include :flash_bar
```

Comment:
Can you move this to after the 'find inbox' text?

Reply:
If I put find inbox under the "Find Inbox", the UI will become a little tight.

Before:
![image](https://user-images.githubusercontent.com/12498220/99899790-05204300-2ce7-11eb-97d9-b94ab77a1d6e.png)
After:
![image](https://user-images.githubusercontent.com/12498220/99899833-2719c580-2ce7-11eb-8069-d289475d5f35.png)

Reply:
Sorry, I meant the big "find inbox" heading. Not the button

Reply:
So it would be like:

Find Inbox

"Error message"

Inbox ID:

Inbox ID input 

Inbox ID submit button